### ISSUE TITLE
安装keras时出现依赖项冲突

### ISSUE THREAD

**Issue by stariveryoung** [bubbliiiing/mask-rcnn-keras#53]
python==3.6
cudatoolkit==10.0
cudnn=7.6.5
tensorflow-gpu==1.13.1
keras==2.1.5
都是在虚拟环境下conda install 的
然后在安装keras的时候出现了图片中显示的错误
<img width="820" alt="Snipaste_2024-05-10_13-11-54" src="https://github.com/bubbliiiing/mask-rcnn-keras/assets/148047092/bd036264-6c5f-4eb7-a499-29f2a66d5ab0">
<img width="850" alt="Snipaste_2024-05-10_13-12-10" src="https://github.com/bubbliiiing/mask-rcnn-keras/assets/148047092/22769b45-f7b3-44bf-8a92-97bcb321931a">

**Comment by stariveryoung** [comment]
导入tensorflows时还出现了这样的错误
![image](https://github.com/bubbliiiing/mask-rcnn-keras/assets/148047092/ce186031-e6cd-4822-aa22-16e1cae460f6)

**Comment by stariveryoung** [comment]
已经解决了，之前一直使用conda install发现很容易出现问题，而对于tensorflow这种包好像用pip install 好一点
